2022/1xxx/CVE-2022-1091.json

```diff
@@ -1,18 +1,80 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2022-1091",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "CVE_data_meta": {
+    "ID": "CVE-2022-1091",
+    "ASSIGNER": "contact@wpscan.com",
+    "STATE": "PUBLIC",
+    "TITLE": "Safe SVG < 1.9.10 - SVG Sanitisation Bypass"
+  },
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "generator": "WPScan CVE Generator",
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Unknown",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Safe SVG",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "1.9.10",
+                      "version_value": "1.9.10"
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "The sanitisation step of the Safe SVG WordPress plugin before 1.9.10 can be bypassed by spoofing the content-type in the POST request to upload a file. Exploiting this vulnerability, an attacker will be able to perform the kinds of attacks that this plugin should prevent (mainly XSS, but depending on further use of uploaded SVG files potentially other XML attacks)."
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "MISC",
+        "url": "https://wpscan.com/vulnerability/4d12533e-bdb7-411f-bcdf-4c5046db13f3",
+        "name": "https://wpscan.com/vulnerability/4d12533e-bdb7-411f-bcdf-4c5046db13f3"
+      },
+      {
+        "refsource": "CONFIRMED",
```

Comment:
This should be "CONFIRM" and not "CONFIRMED"

Reply:
Haaa it's that source, I was looking at

```
"source": {
    "discovery": "EXTERNAL"
  }
```

Will update that in a sec